qutebrowser/misc/editor.py

```diff
@@ -122,7 +122,8 @@ def edit(self, text):
             raise ValueError("Already editing a file!")
         self._text = text
         try:
-            self._oshandle, self._filename = tempfile.mkstemp(text=True)
+            self._oshandle, self._filename = tempfile.mkstemp(text=True,
+                    prefix='qutebrowser_editor_')
```

Comment:
I suggest using `qutebrowser-editor-` instead for consistency, as `qutebrowser-userscript-` is already used for userscripts.

Minor nitpick: please move `text=True` to the next line as well and indent it by 4 spaces, as per [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation).